src/docs/common_tasks/login.md

```diff
@@ -17,18 +17,17 @@ However, in many cases the server expects you to provide credentials to an authe
 in exchange for a session id, and then provide that session id in subsequent requests to the other endpoints.
 
 In the latter case, you can perform that authentication using the `login` goal, and specifying the 
-provider you're authenticating against, either with the `--to` option or with a passthru arg:
+provider you're authenticating against with the `--to` option:
 
 ```bash
 $ ./pants login --to=myauthprovider
-$ ./pants login -- myauthprovider
 ```
 
 This submits credentials from `.netrc` to the provider's authentication endpoint using HTTP basic auth, 
 and stores all returned cookies (which, presumably, include the session id), for future use with the other 
 endpoints.
 
-In the future the `login` goal will optionally prompt for a username and password, allowing you to
+In the future, the `login` goal will optionally prompt for a username and password, allowing you to
```

Comment:
Is there an issue tracking this? May be good to link these two together, so that whoever solves the issue knows to update the docs.

Reply:
No - I've been closely tracking the deprecation of passthru args, though, and now know to update this in the followup changes. Once 1.25.0rc0 goes out, we'll be able to make our last deprecations for old-style passthrough args and we can remove it from the grammar in 1.28.0.dev0!